tests/Jenkins/utilities.sh

```diff
@@ -292,6 +292,11 @@ installDIRAC() {
 # Arguments: list of DiracX submodule module names to install (core, client, etc.)
 
 function installDIRACX() {
+  # Get the latest tag from GitHub
+  latest_tag=$(git ls-remote --tags https://github.com/DIRACGrid/diracx.git | awk -F/ '{print $NF}' | grep -v '{}' | sort -V | tail -n1)
```

Comment:
Using github API is another option:

```suggestion
  latest_tag=$(curl -s "https://api.github.com/repos/DIRACGrid/diracx/releases" | jq -r '.[].tag_name' | sort -V | tail -n 1)
```